src/components/configure/GraphQLTimeout.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import { Form, Input, Button } from 'antd';
+
+const GraphQLTimeout = ({ form, contextTimeout, handleSubmit }) => {
+  const { getFieldDecorator } = form;
+
+  const handleSubmitClick = e => {
+    e.preventDefault();
+    form.validateFields((err, values) => {
+      if (!err) {
+        handleSubmit(values.contextTimeout);
+      }
+    });
+  }
+  return (
+    <div>
+      <p>The timeout of GraphQL requests in seconds</p>
+      <Form>
+        <Form.Item>
+          {getFieldDecorator('contextTimeout', {
```

Comment:
It was supposed to be `contextTime` and not `contextTimeout`. Please change everywhere